packages/suite/src/middlewares/wallet/index.ts

```diff
@@ -1,4 +1,4 @@
-import blockchainMiddleware from './blockchainMiddleware';
+import { blockchainMiddleware } from '@suite-common/wallet-core';
```

Comment:
You are importing two times from @suite-common/wallet-core here:

**import { blockchainMiddleware } from '@suite-common/wallet-core';**
import discoveryMiddleware from './discoveryMiddleware';
**import { prepareFiatRatesMiddleware } from '@suite-common/wallet-core';**
import storageMiddleware from './storageMiddleware';